backend/src/v5/schemas/tickets/index.js

```diff
@@ -62,20 +62,23 @@ const generatePropertiesValidator = async (teamspace, properties) => {
 				}
 
 				if (prop.type === propTypes.ONE_OF) {
-					validator = validator.oneOf(values);
+					validator = validator.oneOf(isNewTicket ? values : values.concat(null));
```

Comment:
it fails with this message

![image](https://user-images.githubusercontent.com/71635617/189957203-d4fdc172-936e-410c-8107-e46cbcdb219e.png)

but I could try to change it to the proper one

Reply:
I'm not talking about the error message, the error message is because you don't append `nullable()` further down if it's isn't required. You're appending `null` as an option here even if the field is an required field, which not be accepting null

I think this line should be 
```suggestion
					validator = validator.oneOf(!isNewTicket && prop.required ? values.concat(null): values);
```


Then your error message problem will also naturally resolve itself.